torch/_inductor/lowering.py

```diff
@@ -5767,6 +5769,93 @@ def resize_storage_bytes_(variable, new_size):
     return variable
 
 
+def resize_impl(self, size, *, memory_format=None):
+    if memory_format == torch.preserve_format:
+        raise RuntimeError(f"unsupported memory format: {memory_format}")
+
+    assert isinstance(self, TensorBox)
+    assert isinstance(size, (list, tuple))
+
+    new_numel = sympy_product(size)
+    old_numel = self.get_numel()
+    dtype = self.get_dtype()
+    device = self.get_device()
+    self_flat = view(self, (-1,))
```

Comment:
I think you need to have `unwrap_view` first in case `self` is only viewing a subset of the full storage.